src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+        for (Ball ball : balls) {
+            System.out.println(ball);
+        }
```

Comment:
```suggestion
        for (int i = 0; i < balls.length; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
Can be simplified to one loop